reproducible/ubuntu/BUILD

```diff
@@ -24,47 +24,39 @@ docker_build(
     cmd = "/bin/bash",
 )
 
-load("//reproducible/ubuntu:nodebootstrap.bzl", "nodebootstrap_image")
+NODE_MAP = {
+    "8_5_0": "dd5ac97074882ba62892bdb91ec5e624c338d772a6218da08c182231031a4446",
+    # "8_6_0": "80551d47440c602c5825a3f6611e13cb0c57d277234dbff693af433a88351a08",
```

Comment:
uncomment 8.6?

Reply:
This was commented as a "stub".  Should we support these two versions of node?  To build both, we would need both tarballs checked in.